libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Generators/LanguageGeneratorManager.cs

```diff
@@ -93,6 +97,28 @@ public static ImportResolverDelegate ResourceExplorerResolver(string locale, Dic
             };
         }
 
+        /// <summary>
+        /// Lazy load generator managet.
+        /// </summary>
+        internal void LazyLoad()
```

Comment:
Consider combining this method with internal void LazyLoad(IEnumerable<Resource> resources) in a single metho